package.json

```diff
@@ -74,6 +74,7 @@
     "@types/node": "^6.0.54",
     "app-root-path": "^2.0.1",
     "glob": "^7.1.1",
+    "moment": "^2.17.1",
```

Comment:
since typeorm is also can be used in browser it should be dependless from other packages to prevent complex configuration in the browser. In short words we cant use moment, we need to write small code that does what you need from moment